src/main/java/com/exadel/controller/EntryController.java

```diff
@@ -94,12 +104,32 @@ public void createEntries(@RequestParam(required = false) Long begin,
     public void updateEntry(@RequestBody EntryDTO entryDTO) {
         Entry entry = entryService.getEntryById(entryDTO.getId());
         entry.updateEntry(entryDTO);
+        if (UserServiceImpl.hasRole(0)) {
```

Comment:
do we need to ask service for current user role? cannot Principal be added as a method parameter?